src/libraries/Microsoft.Extensions.Configuration.Binder/tests/Tests/Microsoft.Extensions.Configuration.Binder.Tests.csproj

```diff
@@ -6,11 +6,16 @@
   </PropertyGroup>
```

Comment:
Instead of `tests\Tests` folder, can this be called `tests\UnitTests`? That would better match other folders. Ex:

https://github.com/dotnet/runtime/tree/main/src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests
https://github.com/dotnet/runtime/tree/main/src/libraries/System.Net.Http/tests/UnitTests
https://github.com/dotnet/runtime/tree/main/src/libraries/System.Text.RegularExpressions/tests/UnitTests